lib/daru/index.rb

```diff
@@ -330,5 +342,12 @@ def values
     def inspect
       "Daru::MultiIndex:#{self.object_id} (levels: #{levels}\nlabels: #{labels})"
     end
+
+    def display input_indexes
```

Comment:
Right above the method. Refer to elsewhere in code to see docs. If you have any questions, see the guides for YARD, that's what we use in daru: http://yardoc.org/